VATSpy.dat

```diff
@@ -8544,7 +8544,7 @@ LSZU|Buttwil|47.264722|8.3025||LSAS|0
 LSZV|Sitterdorf|47.508889|9.262778||LSAS|0
 LSZW|Thun|46.756389|7.600556||LSAS|0
 LSZX|Schaenis|47.171667|9.039444||LSAS|0
-LTAB|Ankara-Guvercinlik|39.93306|32.75||LTAA|0

```

Comment:
Any reason for this?
Why is the name now only "-Guvercinlik" with the - symbol?